README.md

```diff
@@ -20,3 +20,98 @@ cli-kit new myproject
 
 You can see example usage [here](https://github.com/Shopify/cli-kit-example). This app is similar to
 the one that the generator generates.
+
+## Starting a New Project
+To begin creating your first `cli-kit` application, run:
+```bash
+cli-kit new myproject
+```
+Where `myproject` is the name of the application you wish to create.  Then, you will be prompted to
+select how the project consumes `cli-kit` and `cli-ui`.  The available options are:
+- Vendor (faster execution, more difficult to update dependencies)
+- Bundler (slower execution, easier dependency management)
+
+You're now ready to write your very first `cli-kit` application!
+
+## How do `cli-kit` Applications Work?
+The executable for your `cli-kit` app is stored in the "exe" directory.  To execute the app, simply
+run:
+```bash
+./exe/myproject
+```
+
+### Folder Structure
+* `/exe/` - Location of the executables for your application.
+* `/lib/` - Location of the resources for your app (modules, classes, helpers, etc).
+    * `myproject.rb` - The main file for your application.
```

Comment:
We should elaborat eon what main file is. It's sort of the "file to load everything". Lisa is really good at turning words into proper meaning.

Reply:
This file is the starting point for where to look for other files, and if you follow the autoloading conventions of cli-kit is the only file that ever need be `require`d.

Reply:
Should I use this entire sentence? 🙂